recipe/meta.yaml

```diff
@@ -0,0 +1,161 @@
+  missing_dso_whitelist:
+    - lib/libstdc++.so.*            # [linux]
+    - lib/libgcc_s.so.*             # [linux]
+    - lib/libgomp.so.*              # [linux]
+    - $RPATH/ld64.so.1              # [linux and s390x]
+    - x86_64-conda-linux-gnu/sysroot/lib64/libm.so.*        # [linux and x64_64]
+    - x86_64-conda-linux-gnu/sysroot/lib64/libc.so.*        # [linux and x64_64]
+    - x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0  # [linux and x64_64]
+    - s390x-conda-linux-gnu/sysroot/lib64/libc.so.*         # [linux and s390x]
+    # contained in ffmpeg
+    - $RPATH/libavutil.*.dylib      # [osx]
+    - $RPATH/libavcodec.*.dylib     # [osx]
+    - $RPATH/libavformat.*.dylib    # [osx]
+    - $RPATH/libavdevice.*.dylib    # [osx]
+    - $RPATH/libavfilter.*.dylib    # [osx]
+    - $RPATH/libavutil.so.*         # [linux]
+    - $RPATH/libc10.so              # [linux]
+    - $RPATH/libavutil.so.*         # [linux]
+    - $RPATH/libavformat.so.*       # [linux]
+    - $RPATH/libavfilter.so.*       # [linux]
+    # libsox shouldn't be activated to begin with
+    - $RPATH/libsox.dylib           # [osx]
+    # in pytorch package
+    - $RPATH/libtorch.dylib         # [osx]
+    - $RPATH/libtorch_cpu.dylib     # [osx]
+    - $RPATH/libtorch_python.dylib  # [osx]
+    - $RPATH/libtorch_cpu.so        # [linux]
+    - $RPATH/libtorch_python.so     # [linux]
+    - $RPATH/libc10.dylib           # [osx]
+    # in protobuf package
+    - $RPATH/libprotobuf.*.dylib    # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make  # [not win]
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - patchelf  # [linux]
+    - protobuf {{ libprotobuf }}
+    - llvm-openmp  # [osx]
+    - intel-openmp  # [not osx and blas_impl == 'mkl']
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+    - pytorch 2.1.0
+    - pybind11
+    - libabseil
+    - ffmpeg 6.1.1
+  run:
+    - python
+    # according to compatibility matrix in
+    # https://github.com/pytorch/audio/blob/v2.1.2/docs/source/installation.rst#compatibility-matrix
+    - pytorch >=2.1.0,<2.2.0
+    # according to optional dependencies
+    # https://github.com/pytorch/audio/blob/v2.1.2/docs/source/installation.rst#optional-dependencies
+    # in this special case we want users of channel sfe140ed to be able to use the load and save functions, therefore
+    # at least ffmpeg (the mightiest of the backends) is specified as a run dependency.
+    - ffmpeg >=4.4,<7
+    # this is not officially required but it is suggested in some upstream build examples and allows the correct library to be used 
+    - libprotobuf <4.21.0
+  run_constrained:
+    # according to optional dependencies
+    # https://github.com/pytorch/audio/blob/main/docs/source/installation.rst#optional-dependencies
+    # sox has been tested with version 14.4.2, but it is unlikely that other versions will work
+    - sox >=14.4.2
```

Comment:
```suggestion
```
sox is a library C library. Either you compile torchaudio with it in which case it'll be a runtime dep or you don't and it's not a dependency (not even optional).

Reply:
ok, point taken.